src/ProgressOnderwijsUtils/Data/ColumnDefinition.cs

```diff
@@ -52,5 +52,5 @@ static ColumnAccessibility DbColumnMetaDataAccessibility(IDbColumn col)
         };
 
     public static bool ShouldIncludePropertyInSqlInsert(IPocoProperty o)
-        => o.CanRead && o.PropertyInfo.Attr<DatabaseGeneratedAttribute>() is not { DatabaseGeneratedOption: not DatabaseGeneratedOption.None, };
+        => o.CanRead && o.PropertyInfo.Attr<DatabaseGeneratedAttribute>() is not { DatabaseGeneratedOption: DatabaseGeneratedOption.Identity, };
```

Comment:
```suggestion
        => o.CanRead;
```

toch?

Reply:
An identity column is not marked as readonly

Reply:
Maar dit gaat toch over de source columns? Dus waarom zou het uitmaken of de source een identity is?

Reply:
Nou omdat als we deze wel in de source stoppen dan krijgen we er geen foutmelding op zoals bij de computed columns.

Reply:
Of maak ik een denk fout?

Reply:
Stel je hebt een source identity column, en die wil je naar een temp table schrijven precies eigenljk zoals jij met 'n rowversion wou - dan moet ie toch ook in de source voorkomen?

Ik denk dat dit netto precies zo werkt als de rowversion of computed columns: aan de source kan _niet_ filteren, en aan de target kant optioneel achter een vlaggetje.

De oude uitzondering kan dus weg.

Reply:
(Daarnaast gebruiken we in principe geen identities meer, dus er nou speciaal uitzonderingen voor te onderhouden zonder sterke aanwijzing dat 't nodig is, is sowieso niet heel nodig).

En als we de uitzondering wegmaken, kunnen we deze method ook prima inlinen.

Reply:
https://github.com/progressonderwijs/ProgressOnderwijsUtils/pull/1002 dus